deepmd/pt/model/network/init.py

```diff
@@ -0,0 +1,451 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+import math
+import warnings
+from typing import Optional as _Optional
+
+import torch
+from torch import (
+    Tensor,
+)
+
+# Initialization methods copied from torch.nn.init from version v2.3.0
```

Comment:
PyTorch uses the BSD-3-Clause license, which contains

```
1. Redistributions of source code must retain the above copyright
   notice, this list of conditions and the following disclaimer.
```

we should obey the license requirement.

Reply:
Is https://github.com/deepmodeling/deepmd-kit/pull/3773/commits/bfdc38330f73281a8211ba0eba9919b48876fc15 ok?

Reply:
Noticed that 2.3.0 is released in 2024, changed in https://github.com/deepmodeling/deepmd-kit/pull/3773/commits/5f09c92b470f675a9bbbb3ada5e60c031540a0e2